server.py

```diff
@@ -87,32 +94,37 @@ def __init__(self, host, port, m, protocol_class, socket=None):
             socket = self.make_socket()
         self.socket = socket
         self.socket.bind((host, port))
-        print("Serving on %s %s"%self.socket.getsockname())
+        print("Serving on %s %s" % self.socket.getsockname())
         self.socket.listen(1)
         self.m[self.fileno()] = self
 
-    ### Asyncable interface
+    # Asyncable interface
```

Comment:
I used three `#` to set off the block implementing `Asyncable`. I think this is ok. For simple comments I think the `#` is right, but since we're trying to bound a block of code we should either leave the triple `#` or do this:

```python
...
# Asyncable interface
code
code
code
# end Asyncable interface
```
or some such thing.